src/drivers/smmstore/smi.c

```diff
@@ -79,6 +79,10 @@ static uint32_t smmstorev2_exec(uint8_t command, void *param)
 {
 	uint32_t ret = SMMSTORE_RET_FAILURE;
 
+#if CONFIG(DRIVERS_EFI_UPDATE_CAPSULES)
+	smmstore_should_use_full_flash(&command);
```

Comment:
Good idea, this will allow uses other than capsule updates, if needed.

https://github.com/Dasharo/coreboot/compare/86a434f2060f6593f8f14c119f1a523524304b67..73e7ccc46bf165c90bc68c5f3320a087f7de4faf#diff-016e61faa9edc4c013a1e2d5c18a656c11a2196086f6ae31642b85ad8cbb36fbR146